src/actions/types.ts

```diff
@@ -70,6 +70,30 @@ export type DelegateActionParams = {
   delegator?: Address
 }
 
+//https://github.com/OpenZeppelin/openzeppelin-contracts/blob/e5c63635e3508a8d9d0afed091578cc4bb59a9c7/contracts/governance/IGovernor.sol#L142C1-L157C7
```

Comment:
These need to be generalized - so I wouldn't make it quite so specific to this contract. Needs to be the params we'd use for a vote action on _any_ protocol.

Reply:
makes sense!  I'll try and find another implementation or two

Reply:
I think we're good.

[ENS uses tally](https://www.tally.xyz/gov/ens), and[ ENS token uses openzeppelin](https://github.com/ensdomains/governance-contracts/blob/30a1989d89ab6b859b23db5430d345ef84208616/contracts/ENSToken.sol#L8)

Reply:
PHEW!  nevermind!

https://github.com/llamaxyz/llama/blob/main/src/LlamaCore.sol#L434